bigframes/core/blocks.py

```diff
@@ -2564,13 +2564,13 @@ def _get_rows_as_json_values(self) -> Block:
 ),
 T1 AS (
     SELECT *,
-           JSON_OBJECT(
+           TO_JSON_STRING(JSON_OBJECT(
```

Comment:
Do we need an explicit TO_JSON_STRING? Doesn't the change in ibis_types.py::ibis_dtype_to_bigframes_dtype take care of this post read?

```
    if isinstance(ibis_dtype, ibis_dtypes.JSON):
        ...
        return bigframes.dtypes.JSON_DTYPE
```